source/calculus/source/09-PS/05.ptx

```diff
@@ -0,0 +1,361 @@
+        
+        <statement>
+            <p>
+                Let <m>f(x)</m> be an <m>(k+1)</m> times differentiable function on an interval <m>I</m> of <m>a</m>, and let <m>T_k(x)</m>
+                be its <m>k</m>th degree Taylor polynomial centered at <m>x=a</m>. Then for any <m>x</m> in the interval <m>I</m>, 
+                there exists <m>c</m> between <m>a</m> and <m>x</m> such that
+                <me>
+                    R_k(x)=\dfrac{f^{(k+1)}(c)}{(k+1)!}(x-a)^{k+1}.
+                </me>
+                If there exists <m>M_k</m> such that <m>|f^{(k+1)}(x)|\leq M_k</m> for all <m>x</m> in <m>I</m>, then the error in 
+                the approximation <m>f(x)\approx T_k(x)</m> has an upper bound:
+                <me>
+                    |R_k(x)|\leq \dfrac{M_k}{(k+1)!}|x-a|^{k+1}.
+                </me>               
+                
+            </p>
+        </statement>
+    </theorem>
+    
+    <remark>
+        <title>Using Taylor's Theorem</title>
+        <p>
+            The trickiest part to using Taylor's Theorem is calculating <m>M_k</m> to get a bound for the error
+            <m>|R_k(x)|</m> for the approximation <m>f(x)\approx T_k(x)</m>. 
+        </p>
+    </remark>
+
+    <activity>
+        <introduction>
+            <p>
+                Consider the function <m>f(x)=1/x</m> defined on the interval <m>I=[1,2]</m>.
+            </p>
+        </introduction>
+        <task>
+            <statement>
+                <p>
+                    Calculate the derivatives <m>f'(x)</m>, <m>f''(x)</m>, <m>f'''(x)</m>, and <m>f^{(4)}(x)</m>. 
+                </p>
+                <answer>
+                    <p>
+                        <m>f'(x)=-1/x^2</m>, <m>f''(x)=2/x^3</m>, <m>f'''(x)=-6/x^4</m>, <m>f^{(4)}(x)=24/x^5</m>
+                    </p>
+                </answer>
+            </statement>
+        </task>
+        <task>
+            <statement>
+                <p>
+                    Which of the following can we say above the values of <m>|f^{(k)}(x)|</m> on <m>I</m> for <m>k=1,2,3,4</m>?
+                </p>
+                <ol marker="A.">
+                    <li>
+                        <p>
+                            <m>|f'(x)|</m> and <m>|f'''(x)|</m> are increasing, while <m>|f''(x)|</m> and <m>|f''''(x)|</m> are decreasing.
+                        </p>
+                    </li>
+                    <li>
+                        <p>
+                            All are decreasing.
+                        </p>
+                    </li>
+                    <li>
+                        <p>
+                            All are increasing.
+                        </p>
+                    </li>
+                     <li>
+                        <p>
+                            <m>|f'(x)|</m> and <m>|f'''(x)|</m> are decreasing, while <m>|f''(x)|</m> and <m>|f''''(x)|</m> are decreasing.
+                        </p>
+                    </li>
+                </ol>
```

Comment:
I think the fourth derivative notation should be consistent; I think $f^{(4)}(x)$ looks better than $f^{''''}(x)$, but it doesn't matter which is used, I don't think.